src/assets/logo-light.svg

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="360"
+   height="360"
+   id="svg4006"
+   version="1.1"
+   inkscape:version="0.48.5 r10040"
+   sodipodi:docname="logo-dark.svg"
+   inkscape:export-filename="/Users/anthony/dev/readthedocs/guidelines/assets/logo-dark.png"
```

Comment:
I'm still noticing the file path that we wanted to remove on the source images. I was originally just describing removing this line, not necessarily optimizing the files:

```suggestion
```

I see a few below on this PR in the source assets still as well

Reply:
Just to be sure, these are the original source files that I added back. It's just the `inkscape:export-filename` that needs to be removed?

Reply:
Yeah, that was the original issue that was raised

https://github.com/readthedocs/site-community/pull/25#discussion_r784184333

I think this got misinterpreted as "optimize" somewhere along the way.